RecoTracker/DeDx/python/dedxEstimators_cff.py

```diff
@@ -78,3 +80,15 @@
 doAlldEdXEstimators = cms.Sequence(doAlldEdXEstimatorsTask)
 
 fastSim.toReplaceWith(doAlldEdXEstimatorsTask, cms.Task(dedxHarmonic2, dedxPixelHarmonic2))
+
+# use only the strips for Run-3
+from Configuration.Eras.Modifier_run3_common_cff import run3_common
+run3_common.toModify(dedxHitInfo,
+    lowPtTracksEstimatorParameters = cms.PSet(
+        fraction = cms.double(0),
+        exponent = cms.double(-2.0),
+        truncate = cms.bool(False)
+    ),
```

Comment:
```suggestion
    lowPtTracksEstimatorParameters = dict(
        fraction = 0,
        exponent = -2.0,
        truncate = False),
```

have you tried this syntax?

Reply:
like this it gives `IndentationError: unexpected indent`, moving the `lowPtTracksEstimatorParameters` back or the `fraction`,`exponent`, `truncate` one back still gives the indentation error.

Reply:
well I tried this (copied verbatim) on top of your commit:
```diff
iff --git a/RecoTracker/DeDx/python/dedxEstimators_cff.py b/RecoTracker/DeDx/python/dedxEstimators_cff.py
index 99a498bbb43..61201308adb 100644
--- a/RecoTracker/DeDx/python/dedxEstimators_cff.py
+++ b/RecoTracker/DeDx/python/dedxEstimators_cff.py
@@ -84,11 +84,6 @@ fastSim.toReplaceWith(doAlldEdXEstimatorsTask, cms.Task(dedxHarmonic2, dedxPixel
 # use only the strips for Run-3
 from Configuration.Eras.Modifier_run3_common_cff import run3_common
 run3_common.toModify(dedxHitInfo,
-    lowPtTracksEstimatorParameters = cms.PSet(
-        fraction = cms.double(0),
-        exponent = cms.double(-2.0),
-        truncate = cms.bool(False)
-    ),
-    usePixelForPrescales = False
-)
+                     lowPtTracksEstimatorParameters = dict(fraction = 0., exponent = -2.0,truncate = False),
+                     usePixelForPrescales = False)
```
and it works for me (tested with: `runTheMatrix.py -l 11634.0 -j 0`)

Reply:
Thanks, I confirm this works